src/chat/chatter.py

```diff
@@ -387,6 +387,7 @@ def viewReplay(self):
     @QtCore.pyqtSlot()
     def viewVaultReplay(self):
         """ see the player replays in the vault """
+        self.lobby.client.replays.setCurrentIndex(2)  # focus on Online Fault
```

Comment:
`self.lobby.client.mainTabs.setCurrentIndex` switches client tab to the replays tab, it's conceptually different than setting up the replay tab. It's fine the way it is, go ahead and merge it.